modules/packages/LinearAlgebra.chpl

```diff
@@ -2490,6 +2490,372 @@ proc cosm(A: []) throws {
   }
 }
 
+// Gets sums along specified axes. Method for Rectangular matrix.
+proc absSum(A: [?D], param axis=0) where !isCSDom(D) {
+  if (axis != 0 || axis != 1) {
+    throw new LinearAlgebraError("absSum(): axis=0 or axis=1 expected");
+  }
+
+  if axis == 0 then
```

Comment:
<note -- I just deleted my last comment, the code was fine>

Is there a reason to write this as `D.dim(1-axis)` instead of just writing it as `D.dim(0)` and `D.dim(1)`? That is easier to read (at least for me, although others might have a different opinion).

Also, I think you can do `A[.., j]` instead of `A[D.dim(axis),j..j]`.

Reply:
Sure Nikhil, I'll check that. I think there was a reason why I did that, I should have added a comment for that, I'm a bit out of context now since I have added it a while ago, I'll reiterate this method and let you know :)